### PR TITLE
refactor(presentation): new collapsible section component

### PR DESCRIPTION
**Pull request by ranihorev**
This PR includes a couple of changes:
- Refactored the component from class component functional component
- Only use the arrow icon and not the glyphicon
- Add an option to use a plus/minus icon instead of the arrow as it's the recommended way (https://www.nngroup.com/articles/accordion-icons/)

### CODE REVIEW

File: app/scripts/modules/core/src/presentation/main.less [hunk 1]
```diff
@@ -945,14 +945,53 @@ select:invalid {
...
+    margin-right: 4px;
+  }
+
+  &.right {
+    margin-left: 4px;
+  }
+  display: flex;
 }
 
-.rotated-90 {
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/presentation/main.less:952]: make sure to check other repos to see if this is used by, e.g., deck-kayenta or deck-nflx

File: app/scripts/modules/core/src/presentation/main.less [hunk 2]
```diff
@@ -945,14 +945,53 @@ select:invalid {
...
+
+.rotate-m90 {
+  transition: @rotate-transition;
   transform: rotate(-90deg);
 }
 
+.rotate-p90 {
+  transition: @rotate-transition;
+  transform: rotate(90deg);
+}
```
Comment (christopherthielen) [inline app/scripts/modules/core/src/presentation/main.less:973]: What does `m` and `p` stand for?

**Comment by christopherthielen** [comment]
very like.  much nice.  wow ❤️

**Review by christopherthielen — APPROVED** [review]
Looks like there are some build errors.  Otherwise, looks great to me! nice work.

File: app/scripts/modules/core/src/presentation/main.less (hunk 2 above)
Comment (ranihorev) [reply app/scripts/modules/core/src/presentation/main.less:973]: Plus/minus. Is it better to rename to -rotate-90 for the negative classes?

Comment (christopherthielen) [reply app/scripts/modules/core/src/presentation/main.less:973]: ah, I see that now.  What about `cw` and ccw`?  Is that more intuitive than `p` and `m`?

@caseyhebebrand @vigneshm @alanmquach

**Comment by ranihorev** [comment]
> Looks like there are some build errors. Otherwise, looks great to me! nice work.

It's related to the presentation package. Do we publish that package separately?

**Comment by ranihorev** [comment]
This was merged as part of another PR. Closing

**Pull request closed without merging**